samples/node/routes/discovery.js

```diff
@@ -17,19 +17,35 @@ exports.root = function (req, res) {
                 },
                 user_input: [],
                 request: {},
-                label: 'Clear Reported Data',
+                label: {
+                    'en-US': 'Clear Reported Data'
+                },
                 type: 'POST',
                 action_key: 'DIRECT'
             }
         },
         object_types: {
             card: {
-                doc: {href: "https://github.com/vmwaresamples/card-connectors-guide/wiki/Card-Responses"},
+                doc: {href: "https://github.com/vmware-samples/card-connectors-guide/wiki/Card-Responses"},
                 fields: {
                     zip: {capture_group: 1, regex: "([0-9]{5})(?:[- ][0-9]{4})?"}
                 },
+                pollable: true,
                 endpoint: {href: `${base}/cards/requests`}
             }
+        },
+        // Logic for this config not implemented.  Value is present to provide mock config data in discovery.
```

Comment:
It would only take a minor tweak to make this functional:

Change:
```
const zips = req.body.tokens.zip || [];
```

to:

```
const zips = req.body.tokens.zip || (req.body.config.defaultZip ? [req.body.config.defaultZip] : []);
```

Reply:
Yeah, I’ll do it.  Would be nice to have it working.

Reply:
You've implemented it, but left the comment in saying it's not implemented.